packages/magento-customer/components/OrderDetails/OrderDetails.tsx

```diff
@@ -105,255 +43,230 @@ export function OrderDetails(props: OrderDetailsProps) {
+            />
+            <Typography>{number}</Typography>
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Order status' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
+            <Typography>
+              <Trans id='Ordered items' />:{' '}
+              {order_date && dateFormatter.format(new Date(order_date))}
+            </Typography>
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Shipping method' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
+            <Typography>{shipping_method ?? ''}</Typography>
 
             {shipments && shipments.length > 0 && (
               <>
-                <div>{shipments?.[0]?.tracking && shipments?.[0]?.tracking?.[0]?.title}</div>
+                {shipments?.[0]?.tracking && shipments?.[0]?.tracking?.[0]?.title}
                 {shipments?.[0]?.tracking?.[0] && (
                   <TrackingLink {...shipments?.[0].tracking?.[0]} sx={{ padding: '4px 0' }} />
                 )}
               </>
             )}
-          </div>
-        </div>
-
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Payment method' />
-          </OrderDetailTitle>
-          <div>
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Payment method' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
             {payment_methods && payment_methods.length < 1 && (
-              <div>
-                <i>
-                  <Trans id='No payment information' />
-                </i>
-              </div>
+              <Typography>
+                <Trans id='No payment information' />
+              </Typography>
             )}
 
             {payment_methods && payment_methods[0] && (
               <>
-                <div>{payment_methods[0].name}</div>
-
+                <Typography>{payment_methods[0].name}</Typography>
                 {invoices && invoices?.length > 0 && (
-                  <Invoice>
+                  <Box
+                    className={classes.invoice}
+                    sx={(theme) => ({
+                      display: 'flex',
+                      justifyContent: 'flex-start',
+                      alignItems: 'center',
+                      color: theme.palette.primary.main,
+                    })}
```

Comment:
This currently has a color, but you can't click on it. Do we want to remove the color here and create an issue here?